neasqc_wp61/models/quantum/embedder.py

```diff
@@ -0,0 +1,322 @@
+"""
+Embedder
+============
+Module containing the base class for transforming sentences in a dataset into embeddings
+"""
+
+from abc import ABC, abstractmethod
```

Comment:
Canon for imports is usually PEP8 + (PEP 8 is a good start, it's not enough for some things)
- alphabetical order
- import X grouped and go before from X import Y
- https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement

Reply:
Thanks for the comments Ryk! addressing them all right now, I will push them all once you have added comments on the rest of the code and i have addressed those too

Reply:
addressed (this is just for me to keep track of what I have already changed)